domain/schema/model.go

```diff
@@ -454,6 +455,55 @@ CREATE TABLE cloud_container (
 
 CREATE UNIQUE INDEX idx_cloud_container_net_node
 ON cloud_container (net_node_uuid);
+
+CREATE TABLE instance_data (
+    machine_uuid           TEXT PRIMARY KEY,
+    instance_id            TEXT NOT NULL,
+    display_name           TEXT NOT NULL,
+    arch                   TEXT,
+    mem                    INT,
+    root_disk              INT,
+    root_disk_source       TEXT,
+    cpu_cores              INT,
+    cpu_power              INT,
+    availability_zone_uuid TEXT,
+    virt_type              TEXT,
+    CONSTRAINT             fk_machine_machine_uuid
+        FOREIGN KEY        (machine_uuid)
+        REFERENCES         machine(uuid),
+    CONSTRAINT             fk_availability_zone_availability_zone_uuid
+        FOREIGN KEY        (availability_zone_uuid)
+        REFERENCES         availability_zone(uuid)
+);
+
+CREATE TABLE instance_tag (
+    machine_uuid    TEXT NOT NULL,
+    tag             TEXT NOT NULL,
+    PRIMARY KEY (machine_uuid, tag),
+    CONSTRAINT      fk_machine_machine_uuid
+        FOREIGN KEY (machine_uuid)
+        REFERENCES  machine(uuid)
+);
+
+CREATE TABLE lxd_profile (
```

Comment:
Drop this for now. It will be added as part of charms.

Reply:
Dropped. See 2072c788bc.